src/ETISS.cpp

```diff
@@ -645,13 +645,26 @@ void etiss::Initializer::loadIniJIT(std::shared_ptr<etiss::CPUCore> cpu)
         return;
     }
 
+    // check if JIT is set on command line
```

Comment:
please replace most of this function with: "cpu->set(getJIT(etiss::cfg().get( ... )))"